.github/workflows/main.yml

```diff
@@ -23,13 +23,42 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2.3.2
 
-      # runs the ftp deploy action
+      # runs the ftp deploy action ECHomepage
       - uses: SamKirkland/FTP-Deploy-Action@4.0.0
         with:
           server: ec-hormersdorf.de
           username: web4236
           password: ${{ secrets.ftp_password }}
           local-dir: ./ECHomepage/
-          server-dir: /html/EC/
-          exclude: prepos-6.config
+          server-dir: /html/ECHomepage/
```

Comment:
das müsste nach /html/ECHomepage/EC/ also noch einen Ordner tiefer.  da die common aufrufe so aussehen: < href="../common/css/bootstrap.min.css">
(einen Ordner hoch und dann in common rein)

Reply:
Aber die Index.html muss doch in dem Hauptordner sein, sonst funktioniert das auf dem alten rödelserver nicht.

Reply:
Achso du hast das wieder auf echomepage umgenannt sehe ich gerade